sysl2/sysl/sysl_test.go

```diff
@@ -599,3 +601,19 @@ func TestMain2WithDataMultipleRelationships(t *testing.T) {
 		fs, logger, main3)
 	testutil.AssertFsHasExactly(t, memFs, "/Relational-Model.png", "/Object-Model.png")
 }
+
+func TestShowBinaryInfo(t *testing.T) {
+	fakeExit := func(int) {
+		panic("os.Exit called")
+	}
+	patch := monkey.Patch(os.Exit, fakeExit)
+	defer patch.Unpatch()
+	assert.PanicsWithValue(
+		t,
+		"os.Exit called",
+		func() {
+			err := showBinaryInfo(nil)
+			assert.NoError(t, err)
```

Comment:
If `showBinaryInfo` panics, you should test that the flow never arrives here:

```go
t.Error("Shouldn't get here.")
```

But this is moot if you reimplement this feature as a subcommand, as recommended elsewhere.